syncd/FlexCounter.cpp

```diff
@@ -2024,6 +2181,42 @@ void FlexCounter::updateSupportedBufferPoolCounters(
     }
 }
 
+void FlexCounter::updateSupportedTunnelCounters(
+        _In_ sai_object_id_t tunnelRid)
+{
+    SWSS_LOG_ENTER();
+
+    if (m_supportedTunnelCounters.size())
+    {
+        return;
+    }
+
+    uint64_t value;
+    for (int cntr_id = SAI_TUNNEL_STAT_IN_OCTETS; cntr_id <= SAI_TUNNEL_STAT_OUT_PACKETS; ++cntr_id)
```

Comment:
Is it possible to pass ```counterIds``` vector to this update function and iterate through it here instead of using hardcoded values?
Otherwise, if values here are hardcoded, it could cause some issues in case number of tunnel counters change.

For example it is done like that in ```updateSupportedQueueCounters()``` function.

Reply:
Updated